recipe/meta.yaml

```diff
@@ -29,6 +29,8 @@ requirements:
     - python
     - {{ pin_compatible('numpy') }}
     - openjdk
+  run_constrained:
+    - javabridge ==9999999999
 
```

Comment:
Looks correct. To be honest I only did this once, so I'll give @jakirkham a chance to look at it before you merge.

Reply:
Yeah seems reasonable to me too :)